chcemvediet/apps/inforequests/admin.py

```diff
@@ -8,6 +8,30 @@
 from .models import Inforequest, InforequestDraft, InforequestEmail, Branch, Action
 
 
+class BranchInline(admin.TabularInline):
+    model = Branch
+    fields = [
+            u'id_field',
+            u'obligee_field',
+            ]
+
+    def id_field(self, obj):
+        return admin_obj_format(obj, u'{obj.pk}')
+    id_field.short_description = u'id'
+
+    def obligee_field(self, obj):
+        return admin_obj_format(obj.obligee, u'{obj.name}')
+    obligee_field.short_description = u'obligee'
+
+    def get_readonly_fields(self, request, obj=None):
```

Comment:
@martinmacko47 od verzia django 2.1 mozeme pouzit tiez `has_change_permission` + `has_view_permission`. [link](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.has_view_permission)

Reply:
Ak na to nezabudneme, ked budeme upgradovat, mozeme to potom prerobit.